lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -392,6 +396,8 @@ defmodule LivebookWeb.SessionLive.CellComponent do
         <%= if @change_indicator do %>
           <span data-element="change-indicator">*</span>
         <% end %>
+        &nbsp;
+        <span><%= @evaluation_time %> seconds</span>
```

Comment:
I would show anything below 100ms as XYms. :)

Reply:
oh, I've been using this snippet, I think it came from Phoenix or Plug logger, so you can adapt it for your needs here too as well

```elixir
      formatted_time =
         if time > 1000 do
           [time |> div(1000) |> Integer.to_string(), "ms"]
         else
           [Integer.to_string(time), "µs"]
         end
```

perhaps we'd differ between Xs, Xms, and Xµs.